app/src/sharedTest/java/org/oppia/app/administratorcontrols/AdministratorControlsActivityTest.kt

```diff
@@ -180,13 +182,46 @@ class AdministratorControlsActivityTest {
       )
       onView(withId(R.id.administrator_controls_list)).perform(scrollToPosition<RecyclerView.ViewHolder>(2))
       onView(atPositionOnView(R.id.administrator_controls_list, 2, R.id.auto_update_topic_switch)).check(
+        matches(
```

Comment:
`@Inject lateinit var profileTestHelper: ProfileTestHelper` is not needed. Remove this.

Reply:
ProfileTestHelper is needed here to initialize profiles because some test-cases will fail if the profiles are not initialized before they run.

Reply:
ProfileHelper is needed here to initialize profiles before the test-cases run as there are some test-cases which are dependent on profiles.